specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/LongTermRetentionBackups.json

```diff
@@ -356,6 +356,75 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/longTermRetentionServers/{longTermRetentionServerName}/longTermRetentionDatabases/{longTermRetentionDatabaseName}/longTermRetentionBackups/{backupName}/changeaccesstier": {
```

Comment:
>changeaccesstier [](http://example.com/codeflow?start=238&length=16)

How about "changeAccessTier" (casing) here?

Reply:
Ack'ed

Reply:
Hi, we cannot change this because:
1. This is a POST operation, we don't provide any value after this word. 
2. Similar Update or Copy LTRBackup APIs are lower cases. 
3. We already got approval from the ARM team in techsync (you can check with james). 
4. Change this one word will cause a lot of other files to change and it takes time, but we don't have time because we need to catch the release of 0501

Reply:
Actually, I have it changed, thanks for the catch